modules/mobile/spec/controllers/application_controller_spec.rb

```diff
@@ -0,0 +1,133 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative '../rails_helper'
+
```

Comment:
Why is this necessary?

Reply:
I should rename the second one, the first one is the core 'rails_helper', the second is the module's 'rails_helper' which has Rspec configurations specific to this module, e.g. flipping the feature on, creating a signed in IAM user.